usaspending_api/api_docs/api_documentation/removed_endpoint.md

```diff
@@ -0,0 +1,7 @@
+## GET
```

Comment:
Won't this response be returned for both GET and POST depending on the route? Since this is not within the `api_contracts` directory that means it isn't looking for the API Blueprint syntax. I would vote to not specify or specify both GET and POST.

Reply:
Doesn't a method need to be specified to be a valid contract?

Reply:
Probably, but it's already not a valid contract, and not in the contracts folder.

Reply:
Perhaps it would be better to not format the file as an API contract.